pkg/agent/core/ngt/config/config.go

```diff
@@ -46,7 +48,7 @@ func NewConfig(path string) (cfg *Data, err error) {
 	}
 
 	if cfg != nil {
-		cfg.Bind()
+		cfg.GlobalConfig.Bind()
```

Comment:
I think cfg.GlobalConfig nil check is necessary

Reply:
thank you for comment.

`GlobalConfig` is not pointer type in current code.
so I think we need not check nil.